docker/debian-dev/install-brotli.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+install_brotli() {
+    apt-get -qy update
+    apt-get install -y sudo cmake wget unzip
+    local BORTLI_VERSION="1.1.0"
+    wget -q https://github.com/google/brotli/archive/refs/tags/v${BORTLI_VERSION}.zip || exit 1
+    unzip v${BORTLI_VERSION}.zip && cd ./brotli-${BORTLI_VERSION} && mkdir build && cd build || exit 1
+    local CMAKE=$(command -v cmake3 >/dev/null 2>&1 && echo cmake3 || echo cmake) || exit 1
+    ${CMAKE} -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/usr/local/brotli .. || exit 1
+    sudo ${CMAKE} --build . --config Release --target install || exit 1
+    if [ -d "/usr/local/brotli/lib64" ]; then
+        echo /usr/local/brotli/lib64 | sudo tee /etc/ld.so.conf.d/brotli.conf
+    else
+        echo /usr/local/brotli/lib | sudo tee /etc/ld.so.conf.d/brotli.conf
+    fi
+    sudo ldconfig || exit 1
```

Comment:
Here, change all "exit -1" to "exit 1".

I don't know how [the original file in apisix-docker](https://github.com/apache/apisix-docker/blob/d5322d57d58c569d98c225f89a8abd991429ec10/debian/install-brotli.sh) passed the review before, which doesn't fit the shell specifications I know, nor can I pass the ci in apisix

<img width="1108" alt="image" src="https://github.com/user-attachments/assets/82e21799-d534-43e9-9cda-be942ab0736b" />

https://github.com/apache/apisix/actions/runs/15530989377/job/43719749891?pr=12300

Reply:
![CleanShot 2025-06-09 at 17 19 38@2x](https://github.com/user-attachments/assets/e2922570-34e4-457f-bd0b-78bff7585654)

This change won't break any existing logic, because I don't see any place that receives `-1` either.